content/news/2024-07-31-maplibre-newsletter-july-2024/index.md

```diff
@@ -0,0 +1,92 @@
+
+We will again use the email voting system OpaVote for the Governing Board election. Sometimes the OpaVote emails can end up in the spam folder. If you are a Voting Member, it might make sense to explicitly allow emails from noreply@opavote.com as a preparation.
+
+## MapLibre GL JS
+
+The main focus of this month was to push forward the globe view. A lot of effort went into the globe controls, gestures, and refactoring the transform class to be different between the globe and mercator view. This is the final stretch for the globe and we are aiming to release it by the end of this year.
+
+We would like to repeat here our gratitude to Jakub for his amazing work on the globe implementation!
+
+While the changes in globe are not considered breaking changes of the public API, the direction right now is to nevertheless bump the major version and release the globe view feature in MapLibre GL JS v5. This is to avoid unexpected breaking changes in plugins and other external libraries that might expect certain behavior from the internal implementation such as the transform class.
+
+## MapLibre Native
+
+The initial [Vulkan backend](https://github.com/maplibre/maplibre-native/pull/2564) implementation is almost ready to merge! In one month it went from being a proof-of-concept to passing all render tests. We don't have any performance numbers at this point, and it is probably too early for those anyway. To fully exploit the capabilities of Vulkan and use it to tune performance we may need to make some architectural challenges to the renderer. We also still need to figure out how to integrate it with Android and how to distribute it for that platform. Of course, OpenGL ES will still be supported as well. That said, impressive and promising progress was made this month.
+
+We managed to create a WebAssembly build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/), taking advantage of the fact that it is a platform supported by Qt (Note that [Qt for WebAssembly](https://doc.qt.io/qt-6/wasm.html), unlike MapLibre Native, is not available under a permissive license). More info can be found in [this issue](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). There is also an issue for making a [WebAssembly build that does not depend on Qt](https://github.com/maplibre/maplibre-native/issues/2554), but this would require implementing our own platform code for WebAssembly (for things like HTTP requests and an event loop) and a OpenGL ES to WebGL translation layer (or WebGPU backend). So this would be quite an undertaking.
```

Comment:
There are several ways we that can move things forward from here, so maybe let's keep the tone optimistic and keep focus on what we've achieved.

Leveraging Qt for WebAssembly, we managed to create a WebGL1 build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/). More info on this demo can found [here](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). This is an early stage, and we're working in multiple directions (overview [here](https://github.com/birkskyum/maplibre-native-wasm)), so better iterations and builds that doesn't depend on Qt (it's used for HTTP requests and an event loop), will follow. Note that Qt for WebAssembly has a more strict license than MapLibre Native.

Reply:
Sounds good. You have a disclaimer about Qt's license in your README.